tools/topology/sof/pipe-kfbm-capture.m4

```diff
@@ -2,7 +2,7 @@
 #
 # Pipeline Endpoints for connection are :-
 #
-#  host PCM_C <-+- KPBM 0 <-- B1 <-- Volume 0 <-- B0 <-- source DAI0
+#  host PCM_C <-+- KPBM 0 <-- B1 <-- B0 <-- source DAI0
```

Comment:
This doesnt sound right. AFAIK SOF doesnt allow connecting 2 buffers?

Reply:
The Volume has been removed, this is just to update the description to match the topology update 078c3c1a.

Reply:
@yongzhi1 volume couldnt have been removed with removing a buffer too. the connecttion B1 <-- B0 is not possible

Reply:
you are right,  actually line 70 of original file already updated:

host PCM_C <-- B1 <--+- KPBM 0 <-- B0 <-- source DAI0